facdb/sql/moeo_socialservicesitelocations.sql

```diff
@@ -0,0 +1,47 @@
+DROP TABLE IF EXISTS _moeo_socialservicesitelocations;
+
+WITH tmp AS(
+    SELECT MIN(uid) AS uid
+    FROM moeo_socialservicesitelocations
+    GROUP BY program_name||provider_name, address_1
+)
+SELECT
+    uid,
+    source,
+    CONCAT(provider_name, ' ' ,program_name) as facname,
+    parsed_hnum as addressnum,
+    parsed_sname as streetname,
+    address_1 as address,
+    city,
+    postcode as zipcode,
+    borough as boro,
+    LEFT(bin::text, 1) as borocode,
+    bin,
+    bbl,
+    NULL as factype,
+    (CASE
+        WHEN program_name = 'JOBS & INTERNSHIPS' THEN 'Workforce Development'
+        WHEN program_name = 'GERIATRIC MENTAL HEALTH' THEN 'Mental Health'
+        WHEN program_name = 'NORC SITES' THEN 'Senior Services'
+        WHEN program_name = 'READING & WRITING' THEN 'Adult and Immigrant Literacy'
+        WHEN program_name = 'IMMIGRANT SERVICES' THEN 'Immigrant Services'
+        WHEN program_name = 'TRANSPORTATION ONLY' THEN 'Senior Services'
+        WHEN program_name = 'AFTERSCHOOL PROGRAMS' THEN 'After-School Programs'
+        ELSE 'Other Health Care'
+    END) as facsubgrp,
```

Comment:
Need to workshop this so that facsubgrp is more granular since we no longer have a `facsubgrp` input field.  Can we leverage the `service_taxonomy` field?  Explore taking the first value of the `service_taxonomy`.

```
(CASE 
                            WHEN facsubgrp IS NULL THEN
                                CASE
                                WHEN program_name = 'JOBS & INTERNSHIPS' THEN 'Workforce Development'
                                WHEN program_name = 'GERIATRIC MENTAL HEALTH' THEN 'Mental Health'
                                WHEN program_name = 'NORC SITES' THEN 'Senior Services'
                                WHEN program_name = 'READING & WRITING' THEN 'Adult and Immigrant Literacy'
                                WHEN program_name = 'IMMIGRANT SERVICES' THEN 'Immigrant Services'
                                WHEN program_name = 'TRANSPORTATION ONLY' THEN 'Senior Services'
                                WHEN program_name = 'AFTERSCHOOL PROGRAMS' THEN 'After-School Programs'
                                ELSE 'Other Health Care'
                                END
                            ELSE  **logic to get first value of service_taxonomy**
                        END)